.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -51,9 +51,7 @@ Fixes #<Issue Number>
 <!-- This section is to be filled by package maintainer(s) who will
 review this pull request. -->
 
-This checklist is meant to remind the package maintainer(s) who will
-review this pull request of some common things to look for. This list
-is not exhaustive.
+This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.
```

Comment:
What about a line like below?
```
Primary reviewer GitHub name: [edit and fill in]
```

Reply:
I'm somewhat :-1 on this, after all the definition of subpackage maintainers are exactly that they do the review and sign-off on PRs. 
(I find editing descriptions and checklist taking unnecessary time for those doing the workflow without reminder, and working mostly with quick, trivial PRs. 
So, I would aim for the middle ground, keep the list short and focused that ensure those who didn't do these checked on their own would do it. Asking them to fill out even more boxes and edits I feel is even more counter-productive).

Reply:
I'm somewhat :-1 on this, after all the definition of subpackage maintainers are exactly that they do the review and sign-off on PRs.
(I find editing descriptions and checklist taking unnecessary time for those doing the workflow without reminder, and working mostly with quick, trivial PRs.
So, I would aim for the middle ground, keep the list short and focused that ensure those who didn't do these checked on their own would do it. Asking them to fill out even more boxes and edits I feel is even more counter-productive).

Reply:
How about this, I'll let leave this PR open for a few more days to collect opinions. If I get no more opinions in support of this request, I'll _not_ include "primary reviewer" field.

In a way, you can already see who checked what via the GitHub issue edit history, though it takes time to dig through the dropdown menu.

![Screenshot 2021-06-28 160352](https://user-images.githubusercontent.com/2090236/123697035-751d1b80-d82a-11eb-91fd-5e8d1a546cdf.jpg)

Reply:
I'm with @bsipocz here - let's not require editing of the entry - between switching to the review page, then to the bottom of conversation page for CI results and latest discussion, back to the top of conversation page for checklist, and then finally again to the bottom of the page for merging, it is already a bit of a hassle.

Reply:
That's fine to skip the primary review edit. 

It's worth noting however that the reviewer should already be looking at the top of the conversation carefully to check the milestone and labels, and the bottom to make sure CI is OK. So clicking the boxes does not take substantial extra time. Even on trivial ones if a reviewer is too busy for that 15 seconds then I wonder if the review was thorough. (For me, the median time I spend on reviewing PRs is probably more than 15 *minutes*, so 15 seconds at the end is truly in the noise).

Reply:
Fair enough. I certainly have done a few reviews too quickly and had decided independently that it was better to somewhat reduce the number of PRs I reviews (hopefully increasing the quality...).

Reply:
I don't understand what this is changing.  It just looks like it unwrapped this line?  I don't know why it's garnering additional discussion--or is it just that people didn't take a close look at this checklist on the original PR?

Reply:
@embray , the discussion here is about https://github.com/astropy/astropy/pull/11870/files#r660017272 .